lib/modules/manager/woodpecker/extract.spec.ts

```diff
@@ -184,5 +184,44 @@ describe('modules/manager/woodpecker/extract', () => {
         ],
       });
     });
+
+    it('extracts the v.1.0.x version', () => {
+      const res = extractPackageFile(
+        `
+        steps:
+          redis:
+            image: quay.io/something/redis:alpine
+          `,
+        '',
+        {}
+      );
+      expect(res).toEqual({
+        deps: [
+          {
+            depName: 'quay.io/something/redis',
+            currentValue: 'alpine',
+            currentDigest: undefined,
+            replaceString: 'quay.io/something/redis:alpine',
+            autoReplaceStringTemplate:
+              '{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}',
+            datasource: 'docker',
+          },
+        ],
+      });
+    });
+
+    it('return null when no pipeline provided', () => {
+      const res = extractPackageFile(
+        `
+        clone:
+          git:
+            image: woodpeckerci/plugin-git
```

Comment:
wouldn't this also be a valid docker image to extract?

Reply:
To my opinion no

Reply:
https://hub.docker.com/r/woodpeckerci/plugin-git

Reply:
it should be extracted, because you can pin a version and /or digest